scripts/rnaseq2ga.py

```diff
@@ -0,0 +1,271 @@
+"""
+    Script to parse the output file produced by cufflinks.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import sys
+import optparse
+import sqlite3
+
+
+class RNASqliteStore(object):
+    """
+    Defines a sqlite store for RNA data as well as methods for loading the
+    tables.
+    """
+    def __init__(self, rnaQuantDataPath, sqliteFileName=None):
+        # TODO: check to see if the rnaQuantId is in the db and exit if it is
+        # since this is a generator and not an updater
+        if sqliteFileName is not None:
+            sqlFilePath = os.path.join(rnaQuantDataPath, sqliteFileName)
+            if sqliteFileName in os.listdir(rnaQuantDataPath):
+                self._dbConn = sqlite3.connect(sqlFilePath)
+                self._cursor = self._dbConn.cursor()
+            else:
+                self.createNewRepo(sqlFilePath)
+
+    def createNewRepo(self, sqlFilePath):
+        self._dbConn = sqlite3.connect(sqlFilePath)
+        self._cursor = self._dbConn.cursor()
+        self.createTables(self._cursor)
+        self._dbConn.commit()
+
+    def createTables(self, cursor):
+        # annotationIds is a comma separated list
+        cursor.execute('''CREATE TABLE RNAQUANTIFICATION (
+                       id text,
+                       annotation_ids text,
+                       description text,
+                       name text,
+                       read_group_id text)''')
+        cursor.execute('''CREATE TABLE EXPRESSION (
+                       id text,
+                       name text,
+                       rna_quantification_id text,
+                       annotation_id text,
+                       expression real,
+                       feature_group_id text,
+                       is_normalized boolean,
+                       raw_read_count real,
+                       score real,
+                       units text)''')
+
+    def addRNAQuantification(self, datafields):
+        """
+        Adds an RNAQuantification to the db.  Datafields is a tuple in the
+        order:
+        id, annotation_ids, description, name, read_group_id
+        """
+        self._cursor.execute('INSERT INTO RNAQUANTIFICATION VALUES '
```

Comment:
For future reference, I'm thinking of moving my sequence annotation import script to using `executemany` to deal with performance issues.